### PR TITLE
Create tr.yml

### PR DESCRIPTION
**Pull request by Fatih-BaKeR**
Turkish language addition. I hope I can install windows 11 on my old computer and test it :).

### CODE REVIEW

File: tr.yml [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+      newest: "Yeniler"
+      lastUpdated: "Son güncellenenler"
+      alphabeticalOrder: "Alfabetik sıra"
+  settings:
+    language:
+      title: "Dil"
+      description: "Windhawk için tercih ettiğiniz görüntüleme dilini seçin."
+      contribute: "<0>Yeni bir çeviriye katkıda bulunun</0>."
+      credits: "Türkçe çeviri: <0>Fatih Fırıncı</0>."
+      creditsLink: "mailto:john.smith@example.com"
```
Comment (m417z) [inline tr.yml:184]: Please replace with your email or website, or, if you prefer not to have a link, remove the `<0>...</0>` tag from the `credits` string.

File: tr.yml [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+      modVersion: "Mod sürümü"
+      modAuthor:
+        title: "Mod yapımcısı"
+        homepage: "Ana Sayfa"
+        github: "GitHub"
+        twitter: "Twitter"
+      processes:
+        all: "Tüm işlemler"
+        allBut: "All but {{list}}"
+        except: "{{included}} except {{excluded}}"
```
Comment (m417z) [inline tr.yml:100]: These two strings weren't translated.

File: tr.yml [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+        tooltip:
+          target: "Hedef işlem"
+          targets: "Hedef işlemler:"
+          excluded: "Hariç::"
```
Comment (m417z) [inline tr.yml:104]: Two colons by mistake? (`::`)

File: tr.yml [hunk 4]
```diff
@@ -0,0 +1,279 @@
...
+      update: "Güncelle"
+      updateNotNeeded: "Yüklü sürüm, en son sürümle aynı"
+      install: "Yükle"
+      compile: "Derle"
+      disable: "Devre dışı bırak"
+      enable: "Etkinleştir"
+      edit: "Düzenle"
+      fork: "Fork"
```
Comment (m417z) [inline tr.yml:112]: Untranslated on purpose? Perhaps it should be [çatal/çatallanarak](https://tr.wikipedia.org/wiki/%C3%87atal_(yaz%C4%B1l%C4%B1m_geli%C5%9Ftirme)).

File: tr.yml [hunk 5]
```diff
@@ -0,0 +1,279 @@
...
+        description: "Modla ilgili sorunları gidermek için yararlı olabilir."
+        none: "Hiçbiri"
+        modLogs: "Mod günlükleri"
+        detailedLogs: "Ayrıntılı hata ayıklama günlükleri"
+        showLogButton: "Günlük çıktısını göster"
+      modSettings:
+        title: "Mod ayarları"
+        description: "Mod ayarlarınızı kolayca dışa aktarın veya paylaşın."
+        loadButton: "Yükle"
+        loadFormattedButton: "Load formatted"
```
Comment (m417z) [inline tr.yml:144]: Untranslated.

**Review by m417z — COMMENTED** [review]
Thank you! I left several comments.

File: tr.yml (hunk 5 above)
Comment (Fatih-BaKeR) [reply tr.yml]: Thank you very much for seeing and warning my mistakes. I left some for testing and some I translated incorrectly :). Now only the following string is missing: TRAY_EMERGENCY_EXIT: "Signal &service exit. I wonder what that means? Thank you very much. Have a nice day.

Comment (m417z) [reply tr.yml]: "Signal service exit" appears when you hold Shift and right click on the tray icon. It allows to exit Windhawk without admin rights. Perhaps I just need to remove it from the translation, it's a hidden option and it's not that important to have it translated.

**Comment by m417z** [comment]
Thank you! The translation will be available in the next version of Windhawk.

**Pull request merged**